### DISCUSSION TITLE
Question about RK043FN48H LCD 480x272 from Rocktech

### DISCUSSION THREAD

**Discussion by N0ury** [Bodmer/TFT_eSPI#2936] [New processor/display support or capability request]
Is it possible to use the RK043FN48H LCD 480x272 from Rocktech?
Spec are available [here](https://mikrocontroller.bplaced.net/wordpress/wp-content/uploads/2018/01/RK043FN48H-CT672B-V1.0.pdf)
It seems that it uses IC Driver OTA5180A from Orient Display
Spec are available [here](https://www.orientdisplay.com/pdf/OTA5180A.pdf)

Thanks

**Comment by Bodmer** [comment]
No, that drier is not supported. A list of the support is in the library ReadMe.
